pyanaconda/installclasses/fedora_server.py

```diff
@@ -37,6 +37,10 @@ class FedoraServerInstallClass(FedoraBaseInstallClass):
         hidden = True
 
     def setDefaultPartitioning(self, storage):
+        FedoraServerInstallClass.createDefaultPartitioning(storage)
+
+    @staticmethod
+    def createDefaultPartitioning(storage):
```

Comment:
Maybe this method could be provided by `BaseInstallClass`.

Reply:
I don't think it is good idea. This is not the default partitioning. Or anything like that.
If we really want to do something like that then we should create new mixin class for it but I don't know where to put it how to name it and so on and I'm not sure if that would be that beneficial.

Reply:
Ok.